### ISSUE TITLE
vagrant fail to provision the machine 

### ISSUE THREAD

**Issue by john012343210** [SamyPesse/How-to-Make-a-Computer-Operating-System#161]
![image](https://user-images.githubusercontent.com/36737465/80896367-f3158d80-8d1f-11ea-9cf5-1045bc090256.png)

Is it related to the vagrant file ?

MY os: window10

**Comment by ghost** [comment]
i dont think so.you just try again new ubuntu OS.i think this problem just your system is to old!

**Comment by duguxiao** [comment]
您好，感谢您的来信。我将尽快给予您答复，祝好！

LTT
河高

**Comment by dangleenTuple** [comment]
I had the same error. I used this box instead of what it suggested in the tutorial and it works: https://app.vagrantup.com/mrgcastle/boxes/ubuntu-lucid32

vagrant box add mrgcastle/ubuntu-lucid32

vagrant init mrgcastle/ubuntu-lucid32

vagrant up
